website/docs/commands/fmt.md

```diff
@@ -20,29 +20,33 @@ You can check formatting correctness using a `--check` flag:
 scala-cli fmt --check
 ```
 
-### Dialects
+### Scalafmt version and dialect
 
-Scala CLI also supports dialects that are passed to the formatter.
-This value is only used if there is no `.scalafmt.conf` file.
-However, if it exists, then all configuration should be placed there.
-For a list of all possible values, consult
-the [official Scala Dialects documentation](https://scalameta.org/scalafmt/docs/configuration.html#scala-dialects):
+Scala CLI `fmt` command runs `scalafmt` under the hood, which requires `.scalafmt.conf` configuration file with specified **version** and **dialect**. 
```

Comment:
I meant something more along the lines of:
```suggestion
Scala CLI `fmt` command supports passing the dialect and `scalafmt` version directly from the command line, using respectively the `--scalafmt-dialect` and `--scalafmt-version` options. You can even skip passing either of those, which will make Scala CLI to infer a default value. 

If a `.scalafmt.conf` file is present in the workspace, the values will be read from there, unless specified with Scala CLI options.

Otherwise, the default dialect will be inferred based on the configured Scala version (through a Scala CLI option or a using directive), while the default `scalafmt` version will be the latest one used by your Scala CLI version (so it is subject to change when updating Scala CLI).

The Scala CLI `fmt` command runs `scalafmt` under the hood, which *normally* requires `.scalafmt.conf` configuration file with explicitly specified **version** and **dialect** fields. 
The way it is handled by Scala CLI is as follows:
```